src/SFA.DAS.AssessorService.Data/CertificateRepository.cs

```diff
@@ -71,19 +71,45 @@ private bool CheckCertificateData(Certificate certificate, string lastName, Date
 
         public async Task<List<Certificate>> GetCertificates(List<string> statuses)
         {
-           if (statuses == null || !statuses.Any())
+            if (statuses == null || !statuses.Any())
             {
                 return await _context.Certificates
                     .Include(q => q.Organisation)
+                    .AsNoTracking()
                     .ToListAsync();
             }
             else
-           {
-               return await _context.Certificates
-                   .Include(q => q.Organisation)
-                   .Where(x => statuses.Contains(x.Status))
-                   .ToListAsync();
-           }
+            {
+                return await _context.Certificates
+                    .Include(q => q.Organisation)
+                    .Where(x => statuses.Contains(x.Status))
+                    .AsNoTracking()
+                    .ToListAsync();
+            }
+        }
+
+        public async Task<PaginatedList<Certificate>> GetCertificateHistory(Guid organisationId, int pageIndex, int pageSize)
+        {
+            var statuses = new List<string>
+            {
+                CertificateStatus.Submitted,
+                CertificateStatus.Printed,
+                CertificateStatus.Reprint
+            };
+
+            var count = await _context.Certificates
+                .Where(q => q.OrganisationId == organisationId)
+                .CountAsync();          
+          
+            var certificates = await _context.Certificates
+                .Include(q => q.Organisation)
+                .Where(x => statuses.Contains(x.Status) && x.OrganisationId == organisationId)
+                .OrderByDescending(q => q.UpdatedAt)
+                .AsNoTracking()
+                .Skip((pageIndex - 1) * pageSize)
+                .Take(pageSize).ToListAsync();
```

Comment:
As mentioned above, if this can take a username and then JOIN to Contacts.  Dapper it up if you need.

Reply:
yep as before